.github/workflows/main.yml

```diff
@@ -58,3 +58,49 @@ jobs:
           args: release --clean ${{ env.flags }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      # This should have been a matrix, but building separate targets is a paid feature in `goreleaser`
```

Comment:
Could this list be shortened by using glob `warchaeology*` or are the logic that this is also a test that the artifacts actually is built?

Reply:
It is a test that we have all the artifacts that we thought we built yes. If we suddenly stop building artifacts for, let's say Windows, the artifact upload will fail.